src/dysh/spectra/spectrum.py

```diff
@@ -567,6 +566,21 @@ def make_spectrum(cls, data, meta, use_wcs=True, observer_location=None):
         #
         return s
 
+    def __add__(self, other):
+        addition = self.add(other, **{"handle_meta": self._add_meta})
+        addition._target = self._target
```

Comment:
What if other has a _target that does not = self._target?

Reply:
My thinking is that we should preserve the attributes of the first operand when no changes are needed. This because one usually does `sig - ref` and wants the information of the signal.